src/Umbraco.Web.BackOffice/Controllers/ContentController.cs

```diff
@@ -1041,13 +1046,15 @@ private bool ValidateCriticalData(ContentItemSave contentItem, out int variantCo
         /// </remarks>
         private void SaveAndNotify(ContentItemSave contentItem, Func<IContent, OperationResult> saveMethod, int variantCount,
             Dictionary<string, SimpleNotificationModel> notifications, SimpleNotificationModel globalNotifications,
-            string invariantSavedLocalizationAlias, string variantSavedLocalizationAlias, string cultureForInvariantErrors,
+            string savedContentHeaderLocalizationAlias, string invariantSavedLocalizationAlias, string variantSavedLocalizationAlias, string cultureForInvariantErrors,
             out bool wasCancelled)
         {
             var saveResult = saveMethod(contentItem.PersistedContent);
             wasCancelled = saveResult.Success == false && saveResult.Result == OperationResultType.FailedCancelledByEvent;
             if (saveResult.Success)
             {
+                bool isBlueprint = contentItem.PersistedContent.Blueprint;
```

Comment:
Is this line needed? I don't see the variable being used further down?

Reply:
@mikecp yes, you're right. I moved the variable a bit earlier in code. I have removed this again.